### PR TITLE
Fix sh headers

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contrib/relative_path.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```
Comment (yuyichao) [inline contrib/relative_path.sh]: This one is definitely using bash features.

**Comment by StefanKarpinski** [comment]
What kind of system these days doesn't have `bash`?

**Comment by tkelman** [comment]
busybox

**Comment by albap** [comment]
Even if your system uses bash by default, you may wish to run something else, like busybox ash, mksh, etc. Or maybe you're on a BSD and don't want bash as a makedepend.

I think it's always good to keep things portable so they work on any system instead of using obscure bash-only functions.

**Comment by StefanKarpinski** [comment]
I'm not against this, just wondering.

**Pull request closed without merging**